framework/src/play-filters-helpers/src/main/scala/play/filters/csrf/CSRFActions.scala

```diff
@@ -369,6 +369,9 @@ class CSRFActionHelper(
     tokenProvider: TokenProvider
 ) {
 
+  /** Set of Cache-Control header directives that will explicitely prevent response caching. */
+  private val NoCacheDirectives = Set("no-cache", "no-store")
```

Comment:
I think maybe the `private` directive should also be included here:

```suggestion
  private val NoCacheDirectives = Set("no-cache", "no-store", "private")
```

MDN [says this about the `private` directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Cacheability):

> **private**
> Indicates that the response is intended for a single user and must not be stored by a shared cache. A private cache may store the response.

Fastly (the CDN folk) [say this](https://docs.fastly.com/guides/tutorials/cache-control-tutorial#do-not-cache):

> If you want to ensure that a resource is not cached by Fastly, send the following HTTP header with the origin response: `Cache-Control: private`